src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -26,8 +29,10 @@ export class ExamNavigationBarComponent implements OnInit {
     criticalTime = moment.duration(5, 'minutes');
 
     icon: IconProp;
+    getSubmissionForExercise = this.examParticipationService.getSubmissionForExercise;
+    getExerciseButtonTooltip = this.examParticipationService.getExerciseButtonTooltip;
```

Comment:
Here too, I never seen something like this so that's why I'm asking ;)

Reply:
same as above